### ISSUE TITLE
Backticks near the end of line

### ISSUE THREAD

**Issue by borekb** [notaapp/nota#108]
Small thing but I just encountered this situation which seems especially strange in the Hide Markdown mode:

<img width="993" alt="Screen Shot 2021-09-20 at 11 16 28" src="https://user-images.githubusercontent.com/101152/133980079-7f976d17-589a-4e2e-93d3-8ea325fd31d6.png">

<img width="993" alt="Screen Shot 2021-09-20 at 11 16 35" src="https://user-images.githubusercontent.com/101152/133980084-164ad961-9aa0-45e9-8bf8-0469817efc72.png">

**Comment by astoilkov** [comment]
Yeah. Totally agree. We should do something about it. Do you have any ideas on how to fix this?

**Comment by borekb** [comment]
Not really but maybe as a hint, I thought to myself several times already that it's relatively strange to just hide the backtick but leave the space there (in the Hide Markdown mode).

For example, ``` `hello` ``` currently renders as `␣hello␣` while I'd probably expect it to render as just `hello` (no spaces). Maybe this would fix the problem as a by-product.
